### PR TITLE
Send copy of support email to SH19 business team

### PR DESCRIPTION
**Pull request by hepsimo**
- add configuration for SH19 email address
- send copy of support email to SH19 email address

Resolves:
BI-8467

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/efs/api/events/service/SameDayServiceDelayedHandler.java [hunk 1]
```diff
@@ -29,14 +29,18 @@
...
     private EmailService emailService;
     private int supportDelayInMinutes;
+    private String businessEmail;
 
     @Autowired
     public SameDayServiceDelayedHandler(final SubmissionRepository repository,
         final EmailService emailService,
-        @Value("${submission.sameday.support.minutes}") final int supportDelayInMinutes) {
+        @Value("${submission.sameday.support.minutes}") final int supportDelayInMinutes,
+        @Value("${internal.sharecapitalreduction.sh19.sameday.email.address}")
```
Comment (PhilipGlover) [inline src/main/java/uk/gov/companieshouse/efs/api/events/service/SameDayServiceDelayedHandler.java:38]: do you think it ought to say 'business' in the property name to make it clear which one this is for?

Comment (PhilipGlover) [reply src/main/java/uk/gov/companieshouse/efs/api/events/service/SameDayServiceDelayedHandler.java:38]: ah scrap that - it says 'internal', I missed that at the start.

Comment (hepsimo) [reply src/main/java/uk/gov/companieshouse/efs/api/events/service/SameDayServiceDelayedHandler.java:38]: To justify the names:
- `submission.sameday.support.minutes` is the delay when support is notified (and business gets a copy)
- `internal.sharecapitalreduction.sh19.sameday.email.address` is named similar to `internal.sharecapitalreduction.email.address`

Comment (PhilipGlover) [reply src/main/java/uk/gov/companieshouse/efs/api/events/service/SameDayServiceDelayedHandler.java:38]: yeah ok, suppose it just shows that the setup for the different emails isn't trivial.

**Review by PhilipGlover — APPROVED** [review]
have run and delayed SH19 email issued to both addresses:
Screenshot 2021-07-13 at 12 36 23

**Pull request merged**